docs/specification/draft/client/elicitation.mdx

```diff
@@ -151,144 +266,203 @@ To request information from a user, servers send an `elicitation/create` request
 }
 ```
 
-**Reject Response Example:**
+### URL Mode Elicitation Requests
+
+URL mode elicitation enables servers to direct users to external URLs for out-of-band interactions that must not pass through the MCP client. This is essential for auth flows, payment processing, and other sensitive or secure operations.
+
+<Note>
+  **Important**: URL mode elicitation is *not* for authorizing the MCP client's
+  access to the MCP server (that's handled by [MCP
+  authorization](../basic/authorization)). Instead, it's used when the MCP
+  server needs to obtain sensitive information or third-party authorization on
+  behalf of the user. The MCP client's bearer token remains unchanged. The
+  client's only responsibility is to provide the user with context about the
+  elicitation URL the server wants them to open.
+</Note>
+
+URL mode elicitation requests **MUST** specify `mode: "url"` and include these additional parameters:
+
+| Name            | Type   | Description                               |
+| --------------- | ------ | ----------------------------------------- |
+| `url`           | string | The URL that the user should navigate to. |
+| `elicitationId` | string | A unique identifier for the elicitation.  |
+
+The `url` parameter **MUST** contain a valid URL. The `message` parameter **MUST NOT** contain a URL.
+
+#### Example: Request Sensitive Data
+
+This example shows a URL mode elicitation request directing the user to a secure URL where they can provide sensitive information (an API key, for example).
+The same request could direct the user into an OAuth authorization flow, or a payment flow. The only difference is the URL and the message.
+
+**Request:**
 
 ```json
 {
   "jsonrpc": "2.0",
-  "id": 2,
+  "id": 3,
+  "method": "elicitation/create",
+  "params": {
+    "mode": "url",
+    "elicitationId": "550e8400-e29b-41d4-a716-446655440000",
+    "url": "https://mcp.example.com/ui/set_api_key",
+    "message": "Please provide your API key to continue."
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 3,
   "result": {
-    "action": "decline"
+    "action": "accept"
+  }
+}
+```
+
+The response with `action: "accept"` indicates that the user has consented to the
+interaction. It does not mean that the interaction is complete. The interaction occurs out
+of band and the client is not aware of the outcome until and unless the server sends a notification indicating completion.
+
+### Completion Notifications for URL Mode Elicitation
+
+Servers **SHOULD** send a `notifications/elicitation/url/complete` notification when an
```

Comment:
```suggestion
Servers **SHOULD** send a `notifications/elicitation/complete` notification when an
```
nit: does it need to have `url` in it?  if it has the ID the receiver should know what type it is.

Reply:
Thanks @pcarleton, done!